deployment/src/main/resources/dev-ui/qwc-qsp-paths.js

```diff
@@ -32,10 +32,6 @@ export class QwcQspPaths extends LitElement {
     render() {
             return html`
                 <vaadin-grid .items="${paths}" class="paths-table" theme="no-border" all-rows-visible>
-                  <vaadin-grid-column auto-width
```

Comment:
this was pretty useful to also have the template id in dev ui in the case of path

Reply:
You're right, we should also show the original path if a link is used.